etn-ccis/blui-react-workflows
# remove cypress & tools

Fixes BLUI-4502.



#### Changes proposed in this Pull Request:

- remove cypress & tooling
- remove cypress from CI
- update licenses



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- Verify precommit
- Verify codecov report



#### Any specific feedback you are looking for?

-
